google/cloud/forseti/common/util/email/sendgrid_connector.py

```diff
@@ -18,16 +18,15 @@
 # this is needed because quiet the Sendgrid no-member error on Travis.
 # pylint: disable=no-member,useless-suppression
 
+
 import base64
-import os
 import urllib2
 
-import jinja2
-
 import sendgrid
 from sendgrid.helpers import mail
 from retrying import retry
 
+from google.cloud.forseti.common.util.email import base_email_connector
```

Comment:
Should go at the bottom of this grouping.

Reply:
Done